src/js/ClspClient/Router/RouterStreamManager.js

```diff
@@ -300,15 +307,29 @@ export default class RouterStreamManager extends RouterBaseManager {
   async _requestStreamData () {
     this.logger.debug('Requesting Stream...');
 
+    let payload;
+
+    if (this.streamConfiguration.tokenConfig &&
+      this.streamConfiguration.tokenConfig.jwt &&
+      this.streamConfiguration.tokenConfig.jwt.length > 0
```

Comment:
SUGGESTION: might want to put this check into a method since we use it in two places.